pkg/apis/rook.io/v1/storage.go

```diff
@@ -147,3 +147,21 @@ func (s NodesByName) Swap(i, j int) {
 func (s NodesByName) Less(i, j int) bool {
 	return s[i].Name < s[j].Name
 }
+
+// IsOnPVC returns whether the Ceph Cluster will run PVC
+func (s *StorageScopeSpec) IsOnPVC() bool {
+	return len(s.StorageClassDeviceSets) != 0
+}
+
+// IsOnPVCEncrypted returns whether a Ceph Cluster on PVC will be encrypted
+func (s *StorageScopeSpec) IsOnPVCEncrypted() bool {
+	if !s.IsOnPVC() {
```

Comment:
No need for this helper, right? If there are no device sets, the `for` loop will be skipped and you'll return false at the end of the method anyway.

Reply:
Removed.